metricbeat/module/kibana/stats/data.go

```diff
@@ -0,0 +1,77 @@
+package stats
+
+import (
+	"encoding/json"
+
+	"github.com/elastic/beats/libbeat/common"
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstriface"
+)
+
+var (
+	schema = s.Schema{
+		"cluster_uuid": c.Str("cluster_uuid"),
+		"name":         c.Str("name"),
+		"uuid":         c.Str("uuid"),
+		"version": c.Dict("version", s.Schema{
+			"number": c.Str("number"),
+		}),
+		"status": c.Dict("status", s.Schema{
+			"overall": c.Dict("overall", s.Schema{
+				"state": c.Str("state"),
+			}),
+		}),
+		"uptime_in_millis": c.Int("uptime_in_millis"),
```

Comment:
We should follow our metricbeat conventions here for the units: https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html

Reply:
Just for my clarity, you're suggesting that the change would just be a naming switch in this file, correct?

```
"uptime_in_millis": c.Int("uptime_ms"),
```

?

Reply:
a65066bb2

Reply:
I guess this idea wouldn't work:
```
2018-04-23T17:44:15.601-0700    ERROR   schema/schema.go:41     Error on field 'uptime_in_millis': Missing field: uptime_in_millis, Error: Key uptime_ms not found
2018-04-23T17:44:15.601-0700    ERROR   schema/schema.go:41     Error on field 'avg_in_millis': Missing field: avg_in_millis, Error: key avg_ms not found
2018-04-23T17:44:15.601-0700    ERROR   schema/schema.go:41     Error on field 'max_in_millis': Missing field: max_in_millis, Error: Key max_ms not found
```

@ruflin is there a way we can pivot on the naming within the beat module? It would help us avoid some messy work in Kibana around not introducing breaking API changes.